src/Compilers/CSharp/Portable/Symbols/PublicModel/PreprocessingSymbol.cs

```diff
@@ -36,15 +36,12 @@ public override bool Equals(object? obj)
                 return true;
             }
 
-            if (ReferenceEquals(obj, null))
+            if (obj is not IPreprocessingSymbol other)
```

Comment:
this seems wrong.  it will make a VB pp symbol equal to a C# one.

Reply:
The interface slipped by mistake, will fix to only accept the C# one.